homeassistant/components/litejet/light.py

```diff
@@ -9,16 +11,20 @@
     LightEntity,
     LightEntityFeature,
 )
-from homeassistant.config_entries import ConfigEntry
-from homeassistant.core import HomeAssistant
-from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from .const import CONF_DEFAULT_TRANSITION, DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_NUMBER = "number"
 
+if TYPE_CHECKING:
```

Comment:
At this point, we are unsure if this is helpful, so we should not do this until that has been decided on (or, there are situation-specific performance needs for it).